src/main/resources/db/migration/0250/V268__NewSpeciesFields.sql

```diff
@@ -0,0 +1,21 @@
+-- Text fields
+ALTER TABLE species ADD COLUMN ecological_role_known TEXT;
+ALTER TABLE species ADD COLUMN dbh_source TEXT;
+ALTER TABLE species ADD COLUMN height_at_maturity_source TEXT;
+ALTER TABLE species ADD COLUMN local_uses_known TEXT;
+ALTER TABLE species ADD COLUMN native_ecosystem TEXT;
+ALTER TABLE species ADD COLUMN other_facts TEXT;
+
+-- Number fields
+ALTER TABLE species ADD COLUMN average_wood_density FLOAT;
+ALTER TABLE species ADD COLUMN height_at_maturity_value FLOAT;
+ALTER TABLE species ADD COLUMN dbh_value FLOAT;
```

Comment:
These would be better as `NUMERIC` rather than `FLOAT`. `NUMERIC` is an arbitrary-precision type that gets mapped to `BigDecimal` in the application code, and it avoids all the "I saved 56.789 but I got back 56.788999999999" kinds of issues with floating-point numbers.